diagnostic.md

```diff
@@ -56,10 +67,13 @@ let answer = 'not empty'
 
 while (answer !== '') {
   answer = ask("Guess my secret? ")
-  if (answer === 'SeCrEt') {
+  if (answer >= 1 && answer <= 10) {
    console.log("You guessed it!")
    break
```

Comment:
Also, you want to give the user a specific number to guess in addition to checking to see if their answer is between 1-10